File: src/main/java/upbrella/be/rent/entity/ConditionReport.java
package upbrella.be.rent.entity;

import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class ConditionReport {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    @OneToOne
    @JoinColumn(name = "history_id")
    private History history;
    private String content;
    private String etc;
}

File: src/main/kotlin/upbrella/be/rent/entity/ConditionReport.kt
package upbrella.be.rent.entity

import javax.persistence.*

@Entity
class ConditionReport(
    val historyId: Long,
    val content: String? = null,
    val etc: String? = null,
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null
)
